src/model/beneficial.owner.gov.model.ts

```diff
@@ -20,13 +20,14 @@ export const BeneficialOwnerGovKeys: string[] = [
 ];
 
 export interface BeneficialOwnerGov {
-  name?: string
-  principal_address?: Address
-  is_service_address_same_as_principal_address?: yesNoResponse
-  service_address?: Address
-  legal_form?: string
-  law_governed?: string
-  start_date?: InputDate
+  id?: string;
```

Comment:
Is id optional here? Won't one always be set?

Reply:
it should always be set so I'll remove the optional marker.